src/modules/navigator/rtl_params.c

```diff
@@ -110,10 +110,10 @@ PARAM_DEFINE_FLOAT(RTL_MIN_DIST, 5.0f);
  * Fly straight to the return location or planned mission landing and land there or
```

Comment:
@sfuhrer This needs to be updated too. Perhaps don't go into detail - that is what the values are for:

> Return mode destination and flight path (home location, rally point, mission landing pattern, reverse mission).